packages/magiclink-auth-service-api/tests/unit/mapper/magicLinkTokenMapper.test.js

```diff
@@ -0,0 +1,41 @@
+jest.mock('../../../src/util/crypto');
+const magicLinkTokenMapper = require('../../../src/mappers/magicLinkTokenMapper');
+const mockCryptoUtils = require('../../../src/util/crypto');
+const mockMagicLinkData = require('../../resources/magicLinkData.json');
+
+describe('mapper.magicLinkTokenMapper', () => {
+  beforeAll(() => {
+    jest.useFakeTimers('modern');
+    jest.setSystemTime(1629300347);
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  describe('magicLinkDataToToken', () => {
+    it('should create the token payload that has the encrypted magicLinkData as payload', () => {
+      mockCryptoUtils.encrypt.mockReturnValue('mockEncryptedData');
+
+      const tokenPayload = magicLinkTokenMapper.magicLinkDataToToken(mockMagicLinkData);
+
+      expect(tokenPayload).toEqual({
```

Comment:
Like the next test, could be define the comparator to be defined before the comparison occurs? Or the other way round, for consistency.

Reply:
I updated the tests.